ecf/include/head.h

```diff
@@ -4,45 +4,68 @@ export PS4='+ $SECONDS + '
 
 # Variables needed for communication with ecFlow
 export ECF_NAME=%ECF_NAME%
-#export ECF_HOST=%ECF_HOST%
 export ECF_HOST=%ECF_LOGHOST%
 export ECF_PORT=%ECF_PORT%
 export ECF_PASS=%ECF_PASS%
 export ECF_TRYNO=%ECF_TRYNO%
-export ECF_RID=$LSB_JOBID
-
-# Tell ecFlow we have started
-# POST_OUT variable enables LSF post_exec to communicate with ecFlow
-if [ -d /opt/modules ]; then
-    # WCOSS TO4 (Cray XC40)
-    . /opt/modules/default/init/sh
-    module load ecflow
-    POST_OUT=/gpfs/hps/tmpfs/ecflow/ecflow_post_in.$LSB_BATCH_JID
-else
-    # WCOSS Phase 3 (Dell PowerEdge)
-    . /usrx/local/prod/lmod/lmod/init/sh
-    . /gpfs/dell1/nco/ops/nwprod/versions/ecflow_p3.ver
-    module load ips/$ips_ver
-    module load EnvVars/$EnvVars_ver
-    module load ecflow/$ecflow_ver
-    POST_OUT=/var/lsf/ecflow_post_in.$USER.$LSB_BATCH_JID
+export ECF_RID=${ECF_RID:-${PBS_JOBID:-$$}}
+export ECF_JOB=%ECF_JOB%
+export ECF_JOBOUT=%ECF_JOBOUT%
+export ecflow_ver=%ecflow_ver%
+
+if [ -d /apps/ops/prod ]; then # On WCOSS2
+  echo "Running 'module reset'"
+  module reset
+  module load envvar/1.0
+  module load PrgEnv-intel/8.1.0
+  module load craype/2.7.8
+  module load intel/19.1.3.304
 fi
-ecflow_client --init=${ECF_RID}
 
-cat > $POST_OUT <<ENDFILE
-ECF_NAME=${ECF_NAME}
-ECF_HOST=${ECF_HOST}
-ECF_PORT=${ECF_PORT}
-ECF_PASS=${ECF_PASS}
-ECF_TRYNO=${ECF_TRYNO}
-ECF_RID=${ECF_RID}
-ENDFILE
+export HOMEgfs=/lfs/h2/emc/global/noscrub/$USER/git/feature-ops-wcoss2
+. ${HOMEgfs}/versions/run.ver
+export gfs_ver=v16.2
+
+if [ -d /apps/ops/prod ]; then # On WCOSS2
+  export ECF_ROOT=/apps/ops/prod/nco/core/ecflow.v5.6.0.7
+  . ${ECF_ROOT}/versions/run.ver
```

Comment:
I am failing to understand what this is doing.  Why `export`? and why loading `prod_util` in `head.h`